openmetadata-docs/content/v1.6.x-SNAPSHOT/how-to-guides/getting-started/day-1/index.md

```diff
@@ -83,21 +83,35 @@ to import metadata into OpenMetadata on a regular schedule.
 
 - You can repeat these steps to ingest metadata from other data sources as needed.
 
-## Step 3: Invite Users
+## Step 3: Invite Users After Setting Up SMTP
+
+### SMTP Configuration
+
+To invite users you will need to ensure that you have an SMTP server available. With the information for your SMTP server you can configure OpenMetadata to send email alerts by updating the details from the UI.
+
+To update the details from the UI, navigate to Settings > Preferences > Email
+
+{% image
+src="/images/v1.6/how-to-guides/admin-guide/email.png"
+alt="Email Configuration"
+caption="Email Configuration"
+/%}
+
+### Inviting Users
 
 After metadata has been ingested into OpenMetadata, you can [invite users](/how-to-guides/admin-guide/teams-and-users/invite-users) to collaborate on the data and assign different roles.
 
 - Go to **Settings > Team & User Management > Users**.
 
 {% image
-  src="/images/v1.6/getting-started/users.png"
+  src="/images/v1.5/getting-started/users.png"
```

Comment:
Let's revert this change, the images to should refer to the version the content is on. 1.6 content should have images from 1.6

Reply:
ok